lib/vdc/DC.cpp

```diff
@@ -1117,6 +1117,37 @@ vector <int> DC::FileTable::GetEntries() const {
 	return(fds);
 }
 
+bool DC::GetMeshDimLens(string mesh_name, std::vector<size_t> &dims) const {
```

Comment:
I get your point, @StasJ. But non-const references are used pretty much exclusively throughout the VDC API. I think it would be preferable to maintain consistency.

Reply:
Frankly, the use of raw pointers is only considered acceptable in a few very limited scenarios. This is definitely not one of them.

Reply:
Sounds good